7_react/react1/week1/my-app/src/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import "./TodoItem.css"
+
+function TodoItem(props){
+
+    return(
+        <>
+        
+        
+            <ul className="todoItem">&#9733;{props.description},{props.deadline}</ul>
```

Comment:
nice approach with using an object with data! However I think you mixed up `ul` and `li` tags, it should be the other way around :)

Reply:
🤣🤣 OMG yes I did.I will correct it.Thank you so much for the review.